crates/prql-compiler/tests/integration/main.rs

```diff
@@ -155,36 +171,32 @@ impl IntegrationTest for Dialect {
                             })
                             .unwrap(),
                     ),
+                    data_file_root: external_db_default_chinook_dir,
                 })
             }
             _ => None,
         }
     }
-    fn import_csv(&mut self, protocol: &mut dyn DbProtocol, csv_name: &str, runtime: &Runtime) {
-        fn get_path_for_table(csv_name: &str) -> std::path::PathBuf {
-            let mut path = env::current_dir().unwrap();
-            path.extend([
-                "tests",
-                "integration",
-                "data",
-                "chinook",
-                format!("{csv_name}.csv").as_str(),
-            ]);
-            path
-        }
+    fn import_csv(
+        &mut self,
+        protocol: &mut dyn DbProtocol,
+        csv_dir: &str,
+        csv_name: &str,
+        runtime: &Runtime,
+    ) {
         match self {
             Dialect::DuckDb => {
-                let path = get_path_for_table(csv_name);
-                let path = path.display().to_string().replace('"', "");
                 protocol
                     .run_query(
-                        &format!("COPY {csv_name} FROM '{path}' (AUTO_DETECT TRUE);"),
+                        &format!(
+                            "COPY {csv_name} FROM '{csv_dir}/{csv_name}.csv' (AUTO_DETECT TRUE);"
+                        ),
                         runtime,
                     )
                     .unwrap();
             }
             Dialect::SQLite => {
-                let path = get_path_for_table(csv_name);
+                let path = format!("{csv_dir}/{csv_name}.csv");
```

Comment:
Using slashes rather than the path API presumably won't work on windows, but I'm not sure we really mind about that...

Reply:
Generally, slashes work on Windows. It was probably ancient versions that did not work.
https://stackoverflow.com/questions/1589930/so-what-is-the-right-direction-of-the-paths-slash-or-under-windows

For example, it is already used in the following places, but it works, right?
https://github.com/PRQL/prql/blob/f3a750e498aadd86b4ebe168bbfc0ec0b55f7c87/web/book/tests/documentation/website.rs#L16

Reply:
Ah cool! (though the SO answer doesn't seem to say that it works on windows, no?)

Good point re it already working there!